apps/files/src/components/FileItem.vue

```diff
@@ -1,20 +1,57 @@
 <template>
-  <oc-file
-    :name="fileName"
-    :extension="item.extension"
-    :icon="previewIcon"
-    :icon-url="previewUrl"
-    :filename="item.name"
+  <oc-grid
+    gutter="small"
+    class="oc-file uk-flex-inline uk-flex-middle"
     :data-preview-loaded="previewLoaded"
-  />
+  >
+    <oc-img
+      v-if="previewUrl"
+      key="file-preview"
+      class="files-list-file-preview"
+      :src="previewUrl"
+      alt=""
+    />
+    <oc-icon v-else key="file-icon" :name="previewIcon" size="medium" aria-hidden="true" />
+    <div>
+      <div class="file-row-name" :filename="item.name">
+        <span
+          class="uk-text-bold oc-cursor-pointer oc-file-name uk-padding-remove-left"
+          role="button"
+          v-text="fileName"
+        /><span
+          v-if="item.extension"
+          class="uk-text-meta oc-file-extension"
+          v-text="'.' + item.extension"
+        />
+      </div>
+      <div class="uk-flex uk-flex-middle">
+        <translate class="uk-margin-small-right">State:</translate>
```

Comment:
IMO `State` together with indicator icons or with the `-` is not self explanatory. What do you think about `Shares:` or `Shared:` instead?

Reply:
The problem is that `Shared:` is not covering our use case since the indicators can come from an extension and can indicate whatever it wants. We tried to come up together with @micbar with some good way of labelling the indicators and `State:` seems like the easiest and best option for now. @micbar did quick research also on the web on the meaning of words `Status` and `State` and in this case, `State` is the one to go with because `Status` is meant for singular indicator (e.g. `Accepted`)

Reply:
Sounds a lot like we need labels per indicator instead of one generic label.